cli/plan.go

```diff
@@ -54,6 +54,7 @@ func (r *RootCmd) TerraformPlan() *serpent.Command {
 
 			var indented bytes.Buffer
 			_ = json.Indent(&indented, buf.Bytes(), "", "  ")
+			//nolint:gosec
```

Comment:
can we add a description of why we need to disable this lint here?

```suggestion
			//nolint:gosec // I do whatever I want
```